scripts/observers/config.py

```diff
@@ -0,0 +1,113 @@
+import os
+import toml
+import sys
+import shutil
+from pathlib import Path
+from git import Repo
+from dotenv import load_dotenv
+
+load_dotenv()
+
+observer_dir = str(os.getenv('OBSERVER_DIR'))
+METACHAIN = 4294967295
+
+
+def update_toml_indexer(path, shard_id):
+    # prefs.toml
+    data1 = toml.load(path+"/prefs.toml")
+    data1['config']['web-socket']['server-url'] = str(shard_id)
+    if shard_id != METACHAIN:
+        data1['config']['web-socket']['server-url'] = "localhost:" + str(22111+shard_id)
+    else:
+        data1['config']['web-socket']['server-url'] = "localhost:" + str(22111+10)
+    f = open(path+"/prefs.toml", 'w')
+    toml.dump(data1, f)
+    f.close()
+
+
+def update_toml_node(path, shard_id):
+    # prefs.toml
+    data1 = toml.load(path+"/prefs.toml")
+    data1['Preferences']['DestinationShardAsObserver'] = str(shard_id)
+    f = open(path+"/prefs.toml", 'w')
+    toml.dump(data1, f)
+    f.close()
+
+    # external.toml
+    data2 = toml.load(path+"/external.toml")
+    data2['WebSocketConnector']['Enabled'] = True
+    if shard_id != METACHAIN:
+        data2['WebSocketConnector']['URL'] = "localhost:" + str(22111+shard_id)
+    else:
+        data2['WebSocketConnector']['URL'] = "localhost:" + str(22111+10)
+    f = open(path+"/external.toml", 'w')
+    toml.dump(data2, f)
+    f.close()
+
+
+def prepare_observer(shard_id):
+    current_observer = observer_dir + str(shard_id)
+    working_dir_observer = working_dir + current_observer
+    os.mkdir(working_dir_observer)
+    os.mkdir(working_dir_observer + "/indexer")
+    os.mkdir(working_dir_observer + "/node")
+
+    node_config = working_dir_observer + "/node/config"
+    indexer_config = working_dir_observer + "/indexer/config"
+
+    shutil.copytree(config_folder, node_config)
+    shutil.copytree("../../cmd/elasticindexer/config", indexer_config)
```

Comment:
So this is relative to the indexer repository. Understood, can be left as it is for the moment.